src/lib.rs

```diff
@@ -154,6 +154,7 @@ pub mod pylib {
             .expect("Failed to acquire write lock on function registry");
         functions_registry.register::<daft_functions::numeric::NumericFunctions>();
         functions_registry.register::<daft_functions::float::FloatFunctions>();
+        functions_registry.register::<daft_functions::uri::UriFunctions>();
         functions_registry.register::<daft_image::functions::ImageFunctions>();
```

Comment:
Btw, would this be the right place to populate the function registry? `FUNCTION_REGISTRY` would be empty if the Python feature was not enabled

Reply:
we could add a [ctor](https://docs.rs/ctor/latest/ctor/attr.ctor.html) to the `src/lib`, but since our build commands all expect python anyways, this seemed like a reasonable approach. The only time we **need** all of the functions is when we do all of the linking at the end here.